### PR TITLE
feat: Add ssh tunneling

### LINKED ISSUE #124: [Node Books Example] Cleanup Docker FIle
Remove un-needed deps like install vim, npm, node-pre-gyp.

No need to run repolist as well.

Should use ubi8/nodejs-14 instead of ubi8/nodejs-10

### PR DESCRIPTION
**Pull request by abmusse**
Fixes #124

### CODE REVIEW

File: nodejs/express/books/Dockerfile [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
+# Install EPEL https://fedoraproject.org/wiki/EPEL for autossh
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+RUN yum -y install unixODBC-devel gcc autossh
 # odbc driver
-RUN yum -y install unixODBC-devel gcc
-RUN yum -y install python3 && ln -sf python3 /usr/bin/python
-RUN curl -vvv -k 'https://w3.rchland.ibm.com/projects/CA400/Win32/process/express/drivers/pool/v7r2m0/v0001/IBMiAccess_v1r1_LinuxAP_u14.zip' -o ap.zip
-RUN unzip -j ap.zip \*x86_64.rpm
+RUN unzip -j IBMiAccess\*.zip \*x86_64.rpm
 RUN yum -y  install  ./ibm-iaccess-*.rpm
```
Comment (ThePrez) [inline nodejs/express/books/Dockerfile:11]: need to blast ibm-iaccess-32bit*

**Review by ThePrez — CHANGES_REQUESTED** [review]

File: nodejs/express/books/Dockerfile (hunk 1 above)
Comment (kadler) [reply nodejs/express/books/Dockerfile:11]: You wouldn't _need_ to, but there's no reason to pull in all the 32-bit libs that wouldn't be used.

Comment (ThePrez) [reply nodejs/express/books/Dockerfile:12]: They fail to compile because of (IIRC) competing compiler requirements

Comment (abmusse) [reply nodejs/express/books/Dockerfile:11]: Updated to remove 32 bit rpms

**Review by ThePrez — APPROVED** [review]

**Pull request merged**